sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormTrainingClient/FormTrainingClientLiveTests.cs

```diff
@@ -0,0 +1,193 @@
+                Assert.AreEqual(0, doc.Errors.Count);
+            }
+
+            foreach (var subModel in model.Models)
+            {
+                Assert.IsNotNull(subModel.FormType);
+                foreach (var fields in subModel.Fields)
+                {
+                    Assert.IsNotNull(fields.Value.Name);
+                    if (labeled)
+                        Assert.IsNotNull(fields.Value.Accuracy);
+                    else
+                        Assert.IsNotNull(fields.Value.Label);
+                }
+            }
+        }
+
+        [Test]
+        public async Task StartTrainingError()
+        {
+            var client = CreateInstrumentedClient();
+
+            var containerUrl = new Uri("https://someUrl");
+
+            TrainingOperation operation = await client.StartTrainingAsync(containerUrl);
+
+            await operation.WaitForCompletionAsync();
+
+            Assert.IsTrue(operation.HasValue);
+
+            CustomFormModel model = operation.Value;
+
+            Assert.IsNotNull(model.ModelId);
+            Assert.IsNotNull(model.CreatedOn);
+            Assert.IsNotNull(model.LastModified);
+            Assert.IsNotNull(model.Status);
+            Assert.AreEqual(CustomFormModelStatus.Invalid, model.Status);
+            Assert.IsNotNull(model.Errors);
+            Assert.AreEqual(1, model.Errors.Count);
+            Assert.IsNotNull(model.Errors.FirstOrDefault().Code);
+            Assert.IsNotNull(model.Errors.FirstOrDefault().Message);
+        }
+
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task TrainingOps(bool labeled)
+        {
+            var client = CreateInstrumentedClient();
+
+            TrainingOperation operation = await client.StartTrainingAsync(_containerUri, labeled);
+
+            await operation.WaitForCompletionAsync();
+
+            Assert.IsTrue(operation.HasValue);
+
+            CustomFormModel trainedModel = operation.Value;
+
+            CustomFormModel resultModel = await client.GetCustomModelAsync(trainedModel.ModelId);
+
+            Assert.AreEqual(trainedModel.ModelId, resultModel.ModelId);
+            Assert.AreEqual(trainedModel.CreatedOn, resultModel.CreatedOn);
+            Assert.AreEqual(trainedModel.LastModified, resultModel.LastModified);
+            Assert.AreEqual(CustomFormModelStatus.Ready, resultModel.Status);
+            Assert.AreEqual(trainedModel.Status, resultModel.Status);
+            Assert.AreEqual(trainedModel.Errors.Count, resultModel.Errors.Count);
+
+            for (int i=0; i< resultModel.TrainingDocuments.Count; i++)
+            {
+                var tm = trainedModel.TrainingDocuments[i];
+                var rm = resultModel.TrainingDocuments[i];
+
+                Assert.AreEqual(tm.DocumentName, rm.DocumentName);
+                Assert.AreEqual(tm.PageCount, rm.PageCount);
+                Assert.AreEqual(TrainingStatus.Succeeded, rm.Status);
```

Comment:
Same question here - do we know that tm.Status is succeeded?

Reply:
updated